types/linkifyjs/linkifyjs-tests.tsx

```diff
@@ -225,3 +227,17 @@ describe("linkifyjs/react", () => {
         </Linkify>;
     }
 });
+
+describe("linkifyjs/string", () => {
+    const options = {};
```

Comment:
`{}` can fit into a lot of types, so it's possible that someone can incorrectly change the definition of `linkifyStr` and tests could still pass.

As an example, if someone accidentally changed the type signature of `options` to

```diff
- export function linkifyStr(str: string, options?: LinkifyOptions): string;
+ export function linkifyStr(str: string, options?: HTMLElement): string;
```

the test would still pass.

Maybe do this?

```suggestion
    const options = {} as LinkifyOptions;
```

I'm not too sure what the best testing style is, please let me know if the suggestion above feels weird.